astrocut/asdf_cutouts.py

```diff
@@ -0,0 +1,57 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""This module implements cutout functionality similar to fitscut, but for the ASDF file format."""
+
+import asdf
+import astropy
+
+def get_center_pixel(file, ra, dec):
+
+    # Get the roman 2D science image
+    f = asdf.open(file)
+
+    # Get the WCS
+    wcs = f['roman']['meta']['wcs']
+    
+    # Get the WCS header
+    header = wcs.to_fits_sip()
```

Comment:
```suggestion
    # Get the roman 2D science image
    with asdf.open(file) as f:
        # Get the WCS
        wcs = f['roman']['meta']['wcs']

        # Get the WCS header
        header = wcs.to_fits_sip()
```
Using the `with` context here will help to close the opened file.